### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by scpeters**
Use trusty
Set -Werror
make
sudo make install
pkg-config --modversion console-bridge

### CODE REVIEW

**Comment by scpeters** [comment]
@jacquelinekay there's another haiku here

**Comment by scpeters** [comment]
What do you think @j-rivero ?

**Comment by j-rivero** [comment]
> What do you think @j-rivero ?

the haiku is not that bad.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+sudo: required
+dist: trusty
+language: cpp
+compiler:
+  - gcc
+  - clang
+env:
+  - BUILD_TYPE=Debug
+  - BUILD_TYPE=Release
```
Comment (j-rivero) [inline .travis.yml]: In Linux, it is probably more useful to setup RelWithDebInfo instead of just Release. That is the default in debian.

**Pull request merged**